lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -145,4 +155,8 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
             this.trigger.closeMenu();
         }
     }
+
+    isBadgeVisible(): boolean {
```

Comment:
not needed, also the corresponding tests not needed, UI already checks the unreadNotifications length everywhere

Reply:
removed this method and related tests